apps/parsley/README.md

```diff
@@ -17,6 +17,36 @@ Parsley is the UI for Evergreen's log viewer. It is the replacement for
 3. Run `yarn`.
 4. Run `yarn run dev`. This will launch the app.
 
+### Running against a remote Evergreen server
+
+If you want to run Parsley against a remote Evergreen server, you can do so by
+taking the following steps.
+
+1. Generate a self signed certificate and add it to your keychain. This will
+   allow you to run Parsley against a remote Evergreen server that uses HTTPS.
+
+```sh
+brew install mkcert
+mkcert -install
+mkcert -key-file localhost-key.pem -cert-file localhost-cert.pem parsley-local.corp.mongodb.com
```

Comment:
May be worth specifying that this command should be run from within the Parsley directory (and same for Spruce)